scripts/tests_markdown.tpl

```diff
@@ -0,0 +1,138 @@
+{% macro print_statistics_table(has_metrics) -%}
+
+        | | *Value* |
+        |-|-|
+        | **Sum** *(ms)* | ``{{ has_metrics.sum }}`` |
+        | **Median** *(ms)* | ``{{ has_metrics.median }}`` |
+        | **Mean** *(ms)* | ``{{ has_metrics.mean }}`` |
+        | **Standard Deviation** | ``{{ has_metrics.stdev }}`` |
```

Comment:
```suggestion
        | **Sum** *(ms)* | ``{{ has_metrics.sum|sum }}`` |
        | **Median** *(ms)* | ``{{ "%.2f" % has_metrics.median }}`` |
        | **Mean** *(ms)* | ``{{ "%.2f" % has_metrics.mean }}`` |
        | **Standard Deviation** | ``{{ "%.4f" % has_metrics.stdev if has_metrics.stdev is not none else "n/a" }}`` |
```
A very minor tweak for purely aesthetic reasons? I don't think the number of significant digits matters as long as it's consistent, and sum is always an int. Standard deviation can be the same or +2 digits, etc, as long as it's consistent.